python/cudf/cudf/core/series.py

```diff
@@ -1942,7 +1942,9 @@ def sort_index(self, ascending=True):
         inds = self.index.argsort(ascending=ascending)
         return self.take(inds)
 
-    def sort_values(self, ascending=True, na_position="last"):
+    def sort_values(
+        self, ascending=True, na_position="last", ignore_index=False
+    ):
```

Comment:
Align API of this one as well: https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.sort_values.html